anyway/infographics_utils.py

```diff
@@ -245,7 +248,7 @@ def prepare_table(location_info, start_time, end_time):
         )
         # Add casualties
         for accident in accidents:
-            accident["type"] = accident["accident_type"]
+            accident["type"] = accident_type_dict[accident["accident_type"]]
```

Comment:
how can you be certain that the language used to create the dict is the same used here (so you won't get a keyError)?

Reply:
You are right, I can't.